### PR TITLE
remove reference to file that was removed from mbedtls

### PR DESCRIPTION
**Pull request by barbibulle**
When updating the mbedtls 2.x version, on file was removed from the distribution. This removes the reference from the XCode project.

### CODE REVIEW

File: platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2644,7 +2641,6 @@
 		020000000000000000000412 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				BuildIndependentTargetsInParallel = YES;
```
Comment (emixb) [inline platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj:2647]: Is this change intended?

**Review by emixb — APPROVED** [review]

File: platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj (hunk 1 above)
Comment (barbibulle) [reply platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj:2647]: It looks like XCode automatically added that when I saved the project (I didn't change anything explicitly). Do you think that's an issue?

Comment (emixb) [reply platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj:2647]: Xcode likes to clean things up, probably `BuildIndependentTargetsInParallel = YES` is the default setting now, we should be fine

**Review by remusleonte — APPROVED** [review]

**Review by radumvlad — APPROVED** [review]
